### DISCUSSION TITLE
Can this plot only pie charts ?

### DISCUSSION THREAD

**Discussion by aahnik** [mathix420/notion-charts#10] [Q&A] [answered]
First of all Thanks a ton for this awesome tool.

My question is : How can we plot other types of graphs like line or bar chart ?

**Comment by mathix420** [comment]
Glad you like it!

For now it only works with pie charts, but it uses chart.js (quickcharts.io) and google charts which both handle multiple other types of charts.

I plan on integrating many other chart types but I am pretty busy at this time, but if you have the time you can do it yourself in a fork. It's pretty easy, just have a look to google chart doc and quickcharts doc.

**Comment by mathix420** [comment]
Here is a preview of all chart types Notion-Charts can handle

https://quickchart.io/gallery/
https://developers.google.com/chart/interactive/docs/gallery

**Comment by mathix420** [comment]
Hey have a look at this https://notion-charts-git-multicharts-mathix420.vercel.app/
I'm integrating new stuff to handle more charts.

If you want to help you can submit PR on this branch https://github.com/mathix420/notion-charts/tree/multicharts !

**Comment by mathix420** [comment, marked as answer]
Hey, you can now use https://charts.mathix.ninja for multiple charts :rocket:

**Reply by aahnik** [reply]
thank you! love it.
